api/src/models/FedoraObject.test.ts

```diff
@@ -21,7 +21,7 @@ describe("FedoraObject", () => {
         filename = "test1";
         stream = "test2";
         mimeType = "test3";
-        jest.spyOn(fs, "readFileSync").mockReturnValue(buffer);
+        (fs.readFileSync as jest.Mock).mockReturnValue(buffer);
```

Comment:
Looks like the merging didn't undo this unnecessary change. I'll just force it through a review suggestion:

```suggestion
        jest.spyOn(fs, "readFileSync").mockReturnValue(buffer);
```